domain-rrd/rrd-shared/src/main/java/gov/va/vro/model/rrd/mas/MasAnnotation.java

```diff
@@ -46,9 +51,11 @@ public class MasAnnotation {
   @JsonProperty("observation_date")
   private String observationDate;
 
+  // Beginning location of @code{annotVal} on @code{pageNum}
   @JsonProperty("start")
```

Comment:
We are not really using these I do not think and I do not know what exactly they refer to. This guess is a good one except the pages can be multiple.